File: executable/python/axes-ticks.py
import plotly.plotly as py
from plotly.graph_objs import *

py.sign_in('test-runner', '9h29fe3l0x')

trace1 = Scatter(
    x=[0, 1, 2, 3, 4, 5, 6, 7, 8],
    y=[8, 7, 6, 5, 4, 3, 2, 1, 0]
)
trace2 = Scatter(
    x=[0, 1, 2, 3, 4, 5, 6, 7, 8],
    y=[0, 1, 2, 3, 4, 5, 6, 7, 8]
)
data = Data([trace1, trace2])

layout = Layout(
    xaxis=XAxis(
        tick0=0,
        dtick=0.25,
        ticks='outside',
        ticklen=8,
        tickcolor='#000',
        tickwidth=4,
        autotick=False
    ),
    yaxis=YAxis(
        tick0=0,
        dtick=0.25,
        ticks='outside',
        ticklen=8,
        tickcolor='#000',
        tickwidth=4,
        autotick=False
    )
)

fig = Figure(data=data, layout=layout)
plot_url = py.plot(fig, filename='axes-ticks')

File: executable/r/axes-ticks.r
library(plotly)
p <- plotly(username='test-runner', key='9h29fe3l0x')

trace1 <- list(
  x = c(0, 1, 2, 3, 4, 5, 6, 7, 8), 
  y = c(8, 7, 6, 5, 4, 3, 2, 1, 0)
)
trace2 <- list(
  x = c(0, 1, 2, 3, 4, 5, 6, 7, 8), 
  y = c(0, 1, 2, 3, 4, 5, 6, 7, 8)
)

layout <- list(
  xaxis = list(
    autotick = FALSE, 
    tick0 = 0, 
    dtick = 0.25, 
    ticks = "outside", 
    tickcolor = "#000", 
    ticklen = 8, 
    tickwidth = 4
  ), 
  yaxis = list(
    autotick = FALSE, 
    tick0 = 0, 
    dtick = 0.25, 
    ticks = "outside", 
    tickcolor = "#000", 
    ticklen = 8, 
    tickwidth = 4
  )
)

response <- p$plotly(trace0, trace1, kwargs=list(layout=layout, filename="axes-ticks", fileopt="overwrite"))
url <- response$url
filename <- response$filename